Utils/PlatformHelper.cs

```diff
@@ -21,6 +21,9 @@ static class PlatformHelper {
             if (!string.IsNullOrEmpty(windir) && windir.Contains(@"\", StringComparison.Ordinal) && Directory.Exists(windir)) {
                 _current = Platform.Windows;
 
+            } else if (Directory.Exists("/etc/selinux")){   // SELinux enabled, access to /proc/sys/kernel/ostype might fail
+                _current = Platform.Linux;  // We cannot directly use /proc/sys/kernel/ostype to determine an Android Device
+                                              // because of the strict SELinux policies after Lollipop
```

Comment:
```suggestion
            } else if (Directory.Exists("/etc/selinux")) {
                // SELinux enabled, access to /proc/sys/kernel/ostype might fail
                // We cannot directly use /proc/sys/kernel/ostype to determine an Android Device
                // because of the strict SELinux policies after Lollipop
                _current = Platform.Linux;
```